src/appkit.cs

```diff
@@ -21628,6 +21759,10 @@ interface NSWorkspace : NSWorkspaceAccessibilityExtensions {
 		[Async]
 		[Export ("openURLs:withApplicationAtURL:configuration:completionHandler:")]
 		void OpenUrls (NSUrl[] urls, NSUrl applicationUrl, NSWorkspaceOpenConfiguration configuration, [NullAllowed] Action<NSRunningApplication, NSError> completionHandler);
+
+		[Mac (10,16)]
+		[Export ("iconForContentType:")]
+		NSImage IconForContentType (UTType contentType);
```

Comment:
no verb (missing prefix) and suffix might not be needed (implied by parameter name/type)
-> `GetIcon` ?